cmd/update.go

```diff
@@ -42,6 +46,7 @@ func newUpdateCmd() *cobra.Command {
 	}
 	f := cmd.Flags()
 	f.StringVarP(&uc.dest, "destination", "d", ".", "specify the path to the project directory")
```

Comment:
@davidgamero  looks like we already have the destination flag within the update command as well. Can you elaborate on why we need to how the -s addition on update as well?